opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -492,7 +492,7 @@ def to_json(self, indent=4):
         f_span["attributes"] = self._format_attributes(self._attributes)
         f_span["events"] = self._format_events(self._events)
         f_span["links"] = self._format_links(self._links)
-        f_span["resource"] = self._format_attributes(self._resource.attributes)
+        f_span["resource"] = self._format_resource(self._resource)
```

Comment:
@srikanthccv 
Now that I think about it, this is a behavior change but it's actually solving a bug right? Previously, the "schema_url" information was completely missing.